etna/articles/models.py

```diff
@@ -355,6 +364,13 @@ def gallery_has_translations_transcriptions(self):
                 return True
         return False
 
+    @cached_property
+    def featured_highlight_gallery_items(self):
+        """
+        Retrieves the highlights inside of the featured highlight gallery which is selected
+        """
+        return self.featured_highlight_gallery.highlights
```

Comment:
**Question**: What is the return value `type` here? It might be good to add type-hints here and/or a mention in the docstring to prevent guesswork.